src/AppInstallerCLICore/Commands/InstallCommand.cpp

```diff
@@ -107,6 +108,14 @@ namespace AppInstaller::CLI
                 throw CommandException(Resource::String::InvalidArgumentValueError, s_ArgumentName_Scope, { "user"_lis, "machine"_lis });
             }
         }
+
+        if (execArgs.Contains(Args::Type::Locale))
+        {
+            if (!Utility::IsWellFormedBcp47Tag(execArgs.GetArg(Args::Type::Locale)))
+            {
+                throw CommandException(Resource::String::InvalidArgumentValueError, s_ArgumentName_Locale, { "bcp47 language tags"_lis });

```

Comment:
>"bcp47 language tags" [](http://example.com/codeflow?start=109&length=21)

This is not a localization independent string and it wouldn't fit well into this exception constructor anyway.  I think it should just be dropped so the user gets "invalid arg: locale" and can go look at the help on that.